pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
> @rhcarvalho if you want to limit the width of the comments to 69 chars

Where did I say that?!

> (i'm not going to review each commit separately a comment on each commit ;-).

This exaggeration is not funny ;-)
I'm not talking about one character change per commit nor anything alike.



> If you want to make these changes, a separate PR is preferred, presumably fixing this in all types.go

This is the meat of the discussion. I thought @bparees was passing in little changes with his PRs, we discussed this. Perhaps @PI-Victor should not try to do the same with this particular PR, but still it sounded like opening PRs just for that as you're saying was not something that would be well seen.

Thus I'm asking what would be the best approach here.